workers/api-server/src/http/admin.rs

```diff
@@ -225,15 +198,37 @@ impl TypedHandler for AdminOrderMetadataHandler {
         _headers: HeaderMap,
         _req: Self::Request,
         params: UrlParams,
-        _query_params: QueryParams,
+        query_params: QueryParams,
     ) -> Result<Self::Response, ApiServerError> {
         let order_id = parse_order_id_from_params(&params)?;
-        let order = self
+        let order_metadata = self
             .state
             .get_order_metadata(&order_id)
             .await?
             .ok_or(not_found(ERR_ORDER_NOT_FOUND))?;
 
+        let wallet_id = self
+            .state
+            .get_wallet_for_order(&order_id)
+            .await?
+            .ok_or(not_found(ERR_WALLET_NOT_FOUND))?;
+
+        let include_fillable = parse_include_fillable_from_query_params(&query_params)?;
+        let (fillable, price) = if include_fillable {
+            get_fillable_amount_and_price(
+                &order_metadata,
+                &wallet_id,
+                &self.state,
+                &self.price_reporter_job_queue,
+            )
+            .await
+            .map(|(fillable, price)| (Some(fillable), Some(price)))?
+        } else {
+            (None, None)
+        };
+
+        let order = AdminOrderMetadata { order: order_metadata, wallet_id, fillable, price };
```

Comment:
nit: it's cleaner to just build the order metadata and modify it if `include_fillable` is set; the `else { (None, None) }` looks a bit odd, maybe

```rust
        let mut order = AdminOrderMetadata { order: order_metadata, wallet_id, ..Default::default() };
        let include_fillable = parse_include_fillable_from_query_params(&query_params)?;
        if include_fillable {
            let (fillable, price) = get_fillable_amount_and_price(
                &order_metadata,
                &wallet_id,
                &self.state,
                &self.price_reporter_job_queue,
            )
            .await;
            
            order.fillable = fillable;
            order.price = price;
        }
```